doc/index.rst

```diff
@@ -57,3 +55,9 @@ Or from source, by cloning the repo and then running (from the repo root)::
 
    BaseRegTest
    regression_example
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Utilities
+
+   utils
```

Comment:
Should there be an rst file for this?

Reply:
Oh shoot I forgot to include the rst file in git, will push it up.